qutebrowser/utils/jinja.py

```diff
@@ -47,8 +91,9 @@ def get_source(self, _env, template):
         path = os.path.join(self._subdir, template)
         try:
             source = utils.read_file(path)
-        except OSError:
-            raise jinja2.TemplateNotFound(template)
+        except OSError as e:
+            source = html_fallback.replace("%ERROR%", html.escape(str(e)))
+            log.misc.error("The error.html template could not be found" + path)
```

Comment:
The ` + path` seems a bit weird here - shouldn't this be `"could not be found at " + path` or so?